# Just a random test (test-linter)

Please describe the origin of the rule here.


## Rule Details

This rule aims to...

Examples of **incorrect** code for this rule:

```js

// fill me in

```

Examples of **correct** code for this rule:

```js

// fill me in

```

### Options

If there are any options, describe them here. Otherwise, delete this section.

## When Not To Use It

Give a short description of when it would be appropriate to turn off this rule.

## Further Reading

If there are other links that describe the issue this rule addresses, please include them here in a bulleted list.


## Homework

- [ ] GitHub
- [x] Refactor (file names, naming things, docs)
- [ ] Obfuscate numbers and
- [ ] Extra points: Different string obfuscation strategies (and testing of those)
- [ ] Obfuscate something different (e.g. MemberExpression:enter `foo.bar` => `foo["bar"]` -> )